tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -106,71 +118,93 @@ class TelemetryValidatorTest(absltest.TestCase):
+    unvalidated_entities = validator.GetUnvalidatedEntities()
+    self.assertNotIn(_ENTITY_NAME_3, unvalidated_entities)
+    self.assertIn(_ENTITY_NAME_4, unvalidated_entities)
+
   def testTelemetryValidatorDetectsUnknownEntity(self):
     validator = telemetry_validator.TelemetryValidator({}, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_GOOD)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_1, None, 'Unknown entity')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsMissingPoint(self):
     validator = telemetry_validator.TelemetryValidator(
-      _ENTITY_1, 1, _NULL_CALLBACK)
+      _ENTITIES_1, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_MISSING_POINT)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_1, _POINT_NAME_2, 'Missing point')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsMissingPresentValue(self):
     validator = telemetry_validator.TelemetryValidator(
-      _ENTITY_1, 1, _NULL_CALLBACK)
+      _ENTITIES_1, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_MISSING_PRESENT_VALUE)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_1, _POINT_NAME_1, 'Missing present value')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsInvalidState(self):
     validator = telemetry_validator.TelemetryValidator(
-      _ENTITY_2, 1, _NULL_CALLBACK)
+      _ENTITIES_2, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_INVALID_STATE)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_2, _POINT_NAME_3, 'Invalid state: BAD_STATE')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsInvalidNumber(self):
     validator = telemetry_validator.TelemetryValidator(
-      _ENTITY_1, 1, _NULL_CALLBACK)
+      _ENTITIES_1, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_INVALID_NUMBER)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_1, _POINT_NAME_1, 'Invalid number: BAD_NUMBER')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsBooleanAsInvalidNumber(self):
     validator = telemetry_validator.TelemetryValidator(
-      _ENTITY_1, 1, _NULL_CALLBACK)
+      _ENTITIES_1, 1, _NULL_CALLBACK)
     validator.ValidateMessage(_MESSAGE_INVALID_NUMBER_BOOLEAN)
     error = telemetry_error.TelemetryError(
       _ENTITY_NAME_1, _POINT_NAME_1, 'Invalid number: False')
-    self.assertIn(error, validator.validation_errors)
+    self.assertIn(error, validator.GetErrors())
 
   def testTelemetryValidatorDetectsMultipleErrorsInMessage(self):
```

Comment:
can you please leave a blank line to have the distinguishable block between the Arrange. Act. Asset?

Reply:
Done for all tests.